product_docs/docs/biganimal/release/overview/03_security/01_securing_your_cluster.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: "Securing your cluster"
+---
+
+To harden security in you cluster, EDB recommends that you restict all resources to the following requirements:
+
+##### Allowed container ports list 
+
+BigAnimal runs containers using these ports: 
+
+* 5432
+* 9187
+* 8000
+* 9000
+* 9443
+* 9402
+* 10250
+* 9090
+* 8080
+* 8443
+* 2020
+* 3000
+* 6443
+* 9100
+* 9102
+* 9201
+* 8081
+
+##### Allowed service ports list 
+
+BigAnimal runs containers using these services ports:
+
+* 5432
+* 9402
+* 443
+* 8080
+* 9090
+* 3000
+* 8443
+* 9443
+* 9100
+* 9201
+
+##### Allowed AppArmor profiles
+
+BigAnimal uses AppArmor's *runtime/default* default profile. 
+ 
+##### Allowed capabilities 
+
+BigAnimal runs containers with minimum security capabilities of *FOWNER*, which is only required to implement data plane functionality such as logging.
+
+##### Max allowed memory bytes in Kubernetes cluster
+
+BigAnimal runs containers with maximum allowed memory bytes of 6Gi.
+
+##### Recommended Max allowed CPU units in Kubernetes cluster
+
+BigAnimal runs containers with a maximum of two allowed CPU units 2. 
+
+##### Recommended Allowed host paths 
+
+BigAnimal uses the following hotpaths in its containers:
+
+```js
+
+{
+ "paths": [
+   {
+     "pathPrefix": "/var/log",
+     "readOnly": false
+   },
+   {
+     "pathPrefix": "/var/lib/docker/containers",
+     "readOnly": true
+   },
+   {
+     "pathPrefix": "/",
+     "readOnly": true
+   },
+   {
+     "pathPrefix": "/sys",
+     "readOnly": true
+   },
+   {
+     "pathPrefix": "/proc",
+     "readOnly": true
+   },
+   {
+     "pathPrefix": "/var/run/docker.sock",
+     "readOnly": false
+   },
+   {
+     "pathPrefix": "/run/containerd/containerd.sock",
+     "readOnly": false
+   },
+   {
+     "pathPrefix": "/dev",
+     "readOnly": false
+   },
+   {
+     "pathPrefix": "/boot",
+     "readOnly": true
+   },
+   {
+     "pathPrefix": "/lib/modules",
+     "readOnly": false
+   },
+   {
+     "pathPrefix": "/usr",
+     "readOnly": true
+   },
+   {
+     "pathPrefix": "/etc",
+     "readOnly": true
+   }
+ ]
+}
+
+```
+
+## Remediate resources in your cluster
+
+Azure Security Center (ASC) helps assess and enforce BigAnimal's security requirements across the resources used to deploy your clusters.
+
+### To configure the default ASC policy
+
+Perform the following steps in each of your Azure subscriptions:
+
+1. Launch the Azure Policy service in the Azure portal by searching for and selecting the *Policy* service in the search box at the top.
+
+1. Select **Assignments** on the left side of the Azure Policy page. 
+
+1. Select **Assign Policy** from the top of the Policy - Assignments page.
```

Comment:
We can just choose the default policy and update it. Seems we don't need to assign a new one. I will add one screenshot and add it in the ticket.

Reply:
Thanks for you input. incorporated.